examples/assemblyscript/testdata/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "hello-assemblyscript",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "asc assemblyscript.ts --debug -b none -o assemblyscript.wasm -"
```

Comment:
wondering if we should verify the code is compilable in the examples workflow https://github.com/tetratelabs/wazero/blob/main/.github/workflows/examples.yaml

the same goes for rust examples @codefromthecrypt

Reply:
Added the AS build to the examples workflow

Reply:
rust remains a TODO, but not in this PR!